frame/collective/src/tests.rs

```diff
@@ -89,7 +89,7 @@ pub type MaxMembers = ConstU32<100>;
 
 parameter_types! {
 	pub const MotionDuration: u64 = 3;
-	pub const MaxProposals: u32 = 100;
+	pub const MaxProposals: u32 = 512;
```

Comment:
Will revert this, only wanted to see if the benchmarks work.

Reply:
If we revert it then a future developer could accidentally break it.  
`257` could be a middle ground where it would find trivial collisions but still be reasonably fast without `--release` flag.